modules/apps/27-interchain-accounts/host/keeper/msg_server.go

```diff
@@ -0,0 +1,37 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/errors"
```

Comment:
errorsmod is the import alias we use

Reply:
agree, we should avoid shadowing pkgs from std lib